provenance-checker/base/argo-workflows/provenance-check-template.yaml

```diff
@@ -0,0 +1,142 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: provenance-check
+  annotations:
+    thoth-station.ninja/template-version: 0.0.1
+  labels:
+    app: thoth
+    component: provenance-check
+spec:
+  templates:
+    - name: provenance-check
+      inputs:
+        parameters:
+          - name: THOTH_PROVENANCE_CHECKER_JOB_ID
+          - name: THOTH_DOCUMENT_ID
+          - name: THOTH_WHITELISTED_SOURCES
+          - name: THOTH_ADVISER_REQUIREMENTS
+          - name: THOTH_ADVISER_REQUIREMENTS_LOCKED
+          - name: THOTH_ADVISER_METADATA
+          - name: THOTH_LOG_ADVISER
+          - name: THOTH_S3_ENDPOINT_URL
+          - name: THOTH_CEPH_BUCKET_NAME
+          - name: THOTH_CEPH_BUCKET_PREFIX
+          - name: THOTH_DEPLOYMENT_NAME
+      outputs:
+        artifacts:
+          - name: outputdocument
+            path: "/mnt/workdir/{{inputs.parameters.THOTH_DOCUMENT_ID}}"
+            archive:
+              none: {}
+            s3:
+              key: "{{inputs.parameters.THOTH_CEPH_BUCKET_PREFIX}}/\
+              {{inputs.parameters.THOTH_DEPLOYMENT_NAME}}/\
+              provenance-checker/{{inputs.parameters.THOTH_DOCUMENT_ID}}"
+              endpoint: "{{inputs.parameters.THOTH_S3_ENDPOINT_URL}}"
+              bucket: "{{inputs.parameters.THOTH_CEPH_BUCKET_NAME}}"
+              insecure: true
+              accessKeySecret:
+                name: argo-artifact-repository-secrets
+                key: accessKey
+              secretKeySecret:
+                name: argo-artifact-repository-secrets
+                key: secretKey
+      container:
+        name: provenance-check
+        image: quay.io/thoth-station/adviser:v0.6.0-dev
```

Comment:
this should be simply `adviser` as we use images created in the namespace where the workflow runs.

Reply:
Oh, forgot these from testing. Nice catch...